gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -44,6 +44,11 @@
     </module>
     <module name="SuppressWarningsFilter"/> <!-- baseline-gradle: README.md -->
     <module name="SuppressionCommentFilter"/> <!-- baseline-gradle: README.md -->
+    <module name="SuppressionCommentFilter">
```

Comment:
Leaving the first one in allows supporting the default `// CHECKSTYLE:OFF` format. As a legacy support and for cases when you might have a whole method/class that's riddled with checkstyle errors that you don't want to fix yet for whatever reason